lib/action_network_rest/entry_point.rb

```diff
@@ -0,0 +1,22 @@
+module ActionNetworkRest
+  class EntryPoint < Vertebrae::Model
+    def base_path
+      ''
+    end
+
+    def get
+      response = client.get_request base_path
+      response.body
+    end
+
+    def check_authentication
```

Comment:
A helper method that returns a true/false boolean feels like it would be more ergonomic. `authenticated_successfully?`